code/obj/item/grenades.dm

```diff
@@ -979,10 +979,10 @@ PIPE BOMBS + CONSTRUCTION
 
 	attackby(obj/A as obj, mob/user as mob) // adapted from iv_drips.dm
 		if (iscuttingtool(A) && !(src.slashed) && !(src.primed))
+			boutput(user, "You carefully cut [src] open and dump out the contents.")
 			src.slashed = TRUE
 			src.name = "empty [src.name]" // its empty now!
 			src.desc = "[src.desc] It has been cut open and emptied out."
-			boutput(user, "You carefully cut [src] open and dump out the contents.")
```

Comment:
don't see how this changes/fixes anything

Reply:
moving the print before the name change I assume

Reply:
Exactly

Reply:
ahh!